### ISSUE TITLE
Homepage: Change design of the footer

### ISSUE THREAD

**Issue by kathreenriel** [BCcampus/eypd#267]
Change the design of the footer to match the design of the  header 
![footer](https://user-images.githubusercontent.com/16693738/32470836-3096d1e2-c30f-11e7-9614-fadc97fc4f22.png)

**Comment by alexpflores** [comment]
@kathreenriel Waiting on video screenshot graphic

**Comment by kathreenriel** [comment]
Robyn said that she may not be able to get the video graphic to us by this week.

**Comment by alexpflores** [comment]
PR #310 was merged, ready for validation

**Comment by kathreenriel** [comment]
This is looking super. Changes requested in pic below!
<img width="963" alt="footer" src="https://user-images.githubusercontent.com/16693738/32960688-c607e7ca-cb7a-11e7-9634-21932ef42067.png">

**Comment by bdolor** [comment]
Support and Policies are menus under your control in the admin interface.

**Comment by kathreenriel** [comment]
Okay will take a look and make changes.

**Comment by kathreenriel** [comment]
Tried adding a new menu and  resorting menu items but it doesn't seem to work
<img width="1098" alt="screen shot 2017-11-17 at 10 19 24 am" src="https://user-images.githubusercontent.com/16693738/32962376-1061dcd0-cb81-11e7-8c2e-eaa032c14d5d.png">

**Comment by bdolor** [comment]
one more step. Go to Appearance -> Widgets and look for Footer Middle, Right. Change the menu that displays there: 

![image](https://user-images.githubusercontent.com/2048170/32962514-9d38bf02-cb81-11e7-8125-bcf596aa1c19.png)

**Comment by kathreenriel** [comment]
Thanks for showing the extra step. Made the changes!

**Comment by kathreenriel** [comment]
Okay what could be done was done! Yeh! Will create a new issue for Robyn's consideration.

**Comment by paulagaube** [comment]
Looked in IE11 and the Footer looks good. 
![2017-nov-24-ie11-eypd bccampus ca-footer](https://user-images.githubusercontent.com/26391359/33225362-b453f586-d12a-11e7-80c5-a1c9c757b9fb.png)

**Comment by kathreenriel** [comment]
@paulagaube great

**Comment by paulagaube** [comment]
However, there is a minor difference with the City/Community/Town drop down alignment in IE11.
![2017-nov-24-ie11-eypd bccampus ca-top-city](https://user-images.githubusercontent.com/26391359/33225370-df9fc86e-d12a-11e7-85d2-01b0699782e0.png)
